app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsViewModel.kt

```diff
@@ -20,7 +21,8 @@ import org.oppia.android.domain.oppialogger.OppiaLogger
 import org.oppia.android.domain.profile.ProfileManagementController
 import org.oppia.android.util.data.AsyncResult
 import org.oppia.android.util.data.DataProviders.Companion.toLiveData
-import javax.inject.Inject
+import org.oppia.android.util.platformparameter.EnableEditAccountsOptionsUi
+import org.oppia.android.util.platformparameter.PlatformParameterValue
```

Comment:
You will need to reorganise these to pass the lint checks.
`import javax.inject.Inject` will come at the bottom.

Reply:
PTAL @rt4914 Thanks!